features/checkout/seeing_order_summary/seeing_current_prices_of_products_after_catalog_promotion_becomes_ineligible.feature

```diff
@@ -0,0 +1,23 @@
+@checkout
+Feature: Seeing current prices of products after catalog promotion becomes ineligible
+    In order to buy products in its correct prices
+    As a Customer
+    I want to have products with its current prices in the cart
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "T-Shirt"
+        And this product has "PHP T-Shirt" variant priced at "$20.00" in "United States" channel
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a catalog promotion "Winter sale" available in "United States" channel that reduces price by "25%" and applies on "PHP T-Shirt" variant
+        And I am a logged in customer
+
+    @ui @api
+    Scenario: Processing order with valid prices after catalog promotion becomes ineligibly
+        Given I have "PHP T-Shirt" variant of this product in the cart
+        When the administrator makes this catalog promotion unavailable in the "United States" channel
```

Comment:
I'm still more for something like `And the "Winter sale" catalog promotion is no longer available`, as the Customer has no info about neither administrator's actions nor the concept of _current channel_ 🚀